### PR TITLE
Add citeproc-py

### PR DESCRIPTION
**Pull request by maxb2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maxb2** [comment]
@conda-forge-admin, please ping team

**Comment by maxb2** [comment]
I'm willing to be listed as a maintainer.

File: recipes/citeproc-py/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/citeproc-py-{{ version }}.tar.gz
+  sha256: d9e3a224f936fe2e5033b5d9ffdacab769cedb61d96c4e0cf2f0b488f1d24b4e
+
+build:
+  skip: true  # [py2k]
```
Comment (BastianZim) [inline recipes/citeproc-py/meta.yaml:13]:
```suggestion
```
Cf builds only python >=3.7

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one small thing, then this is good to go.

File: recipes/citeproc-py/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - nose
+    - pip
+
+about:
+  home: https://github.com/brechtm/citeproc-py
+  summary: Citations and bibliography formatter
+  license: 
+    - BSD-2-Clause
+    - CC-BY-SA-3.0
+    - MIT
```
Comment (BastianZim) [inline recipes/citeproc-py/meta.yaml:45]: This need to be included using SPDX expressions: https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

```suggestion
  license: BSD-2-Clause AND CC-BY-SA-3.0 AND MIT
```

Comment (maxb2) [reply recipes/citeproc-py/meta.yaml]: Ah, thanks!

**Review by BastianZim — APPROVED** [review]

**Comment by maxb2** [comment]
The license file for the [citeproc/data/locales](https://github.com/citation-style-language/locales/tree/bd8d2dbc85713b192d426fb02749475df30f0d2c) submodule isn't included in the pypi source package. Should I rerun grayskull with the github source?

**Comment by BastianZim** [comment]
No you can just manually include the file, that's fine. Just add it to the citeproc-py folder and change the path to the name only.

**Comment by maxb2** [comment]
@BastianZim thanks for all the help!

**Pull request merged**